python3/unittest/test_usb_scan.py

```diff
@@ -82,24 +76,23 @@ def __init__(self, devices, interfaces):
         self.interfaces = interfaces
 
     def list_devices(self, **kwargs):
-        if kwargs.pop("subsystem") == "usb":
-            dev_type = kwargs.pop("DEVTYPE")
-            if dev_type == "usb_device":
-                return MocEnumerator(self.devices)
-            elif dev_type == "usb_interface":
-                return MocEnumerator(self.interfaces)
-        return MocEnumerator([])
+        assert kwargs.pop("subsystem") == "usb"
```

Comment:
I think this changes the behavior when
`kwargs.pop("subsystem")` equals `"usb"`
A `MocEnumerator([])` will not be returned, intsead, an `AssertionError` will be raised, is it expected?

Reply:
This change is due to codecov finding that MocEnumator([]) was never executed.

Reply:
Yes, I see now `list_devices` is only called with `subsystem` as `"usb"`, I agree this change make sense and we can update the test code if the `list_devices` is used with other value for parameter `subsystem` later.
Thank you~